cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -444,7 +453,8 @@ async def get_trustworthiness_score_async(
         response: Union[str, Sequence[str]],
     ) -> Union[float, List[float]]:
         """Asynchronously gets trustworthiness score for prompt-response pairs.
-        This method is similar to the [`get_trustworthiness_score()`](#method-get_trustworthiness_score) method but operates asynchronously.
+        This method is similar to the [`get_trustworthiness_score()`](#method-get_trustworthiness_score) method but operates asynchronously,
+        allowing for non-blocking concurrent operations.
```

Comment:
there's a lot of jargon here but could we add one specific scenario where one would prefer the _async version of the analogous regular method?